### ISSUE TITLE
Rework AWS keys config.

### ISSUE THREAD

**Issue by squaresurf** [verypossible/libcluster_ecs#7]
There is most likely an easier way to manage AWS keys than to pass it through the config. This is an issue to remind us to revisit that.

**Comment by danielspofford** [comment]
Yeah if we can just get this off the instance like you said that would be 10/10.
